package.json

```diff
@@ -47,6 +54,7 @@
     "eslint-config-airbnb-base": "^13.1.0",
     "eslint-plugin-import": "^2.14.0",
     "mocha": "^5.2.0",
+    "nodemon": "^1.18.9",
```

Comment:
Can you move nodemon to dev dependencies since it's not used on production? Sorry I missed it earlier

Reply:
Am sorry but I think it's in devDependencies? I saw it when I expanded the file